app/services/unsubscriber.rb

```diff
@@ -0,0 +1,8 @@
+module Unsubscriber
```

Comment:
For such a simple module, I wonder if we can just make this is a method on `Subscriber`.